packages/freezed/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=2.17.0 <3.0.0'
 
 dependencies:
-  analyzer: ^5.0.0
+  analyzer: ">=4.2.0 <6.0.0"
```

Comment:
Did some extensive tests, and the lowest constraint, where the test still succeed is:
`analyzer: ">=4.6.0 <6.0.0"`